src/wrapper.ts

```diff
@@ -31,8 +32,12 @@ let isTraceInitialized = false;
 
 const externalInstrumentations = [];
 
-function requireIfAvailable(names: string[]) {
-  names.forEach((name) => safeRequire(name));
+function requireIfAvailable(names: string[], instrumentedModules: Set<string>) {
```

Comment:
This passing instrumentedModules as parameter and using side effects? Return a list built with a filter (or something like that) and add them all to `INSTRUMENTED_MODULES`